pulp-glue/pulp_glue/common/openapi.py

```diff
@@ -127,6 +127,7 @@ def __init__(
         self.base_url: str = base_url
         self.doc_path: str = doc_path
         self.safe_calls_only: bool = safe_calls_only
+        self.auth_provider: t.Optional[AuthProviderBase] = None
```

Comment:
```suggestion
        self.auth_provider = auth_provider
```
At this point we can just assign the passed parameter (it has a default already) and delete the same assignment 6 lines down.

Reply:
True!